core/class_db.cpp

```diff
@@ -322,6 +322,8 @@ StringName ClassDB::get_parent_class(const StringName &p_class) {
 	OBJTYPE_RLOCK;
 
 	ClassInfo *ti = classes.getptr(p_class);
+	if (!ti)
+		print_line("error");
```

Comment:
If it's going to print an error, it should be a little more descriptive than this.

Reply:
Oh, whoops. That was just some debug code I wrote. Forgot to delete it.

Reply:
Just updated it @YeldhamDev.